libs/core/src/lib/pagination/pagination.component.ts

```diff
@@ -257,6 +257,8 @@ export class PaginationComponent implements OnChanges, OnInit, OnDestroy {
         }
         this._refreshPages();
 
+        this._liveAnnouncer.announce('Page ' + page + ', current page');
```

Comment:
I do not think we should use the liveAnnouncer here, I think instead of doing this we need to add an `aria-selected="true"` property to the currently selected page's HTML so the screen reader will say 'page 3, link, selected'.  That will also help with language issues, the above solution will only work in for English websites

Reply:
I did look into `aria-selected` initially, but following the [w3 page](https://www.w3.org/TR/wai-aria-1.1/#aria-current), we currently use `aria-current` that is supposed to be added to navigation/pagination elements and not `aria-selected`(which announces 'Selected'). We are also using the `<nav>` element which [does not support `aria-selected`](https://www.w3.org/TR/wai-aria-1.2/#navigation), so i don't know if it is okay to use `aria-current` and `aria-selected` both.

Reply:
@mikerodonnell89 please share your thoughts on the above comment, thanks!

Reply:
@mikerodonnell89  using `$localize` in typescript now. Verified that on running `ng extract-i18n`, it extracts the localize id in `messages.xlf`. 
<img width="808" alt="Screenshot 2021-08-09 at 3 17 42 PM" src="https://user-images.githubusercontent.com/53509521/128697792-e1f8c0e8-b026-4e7c-9bbf-7ccb96ba20c5.png">

Reply:
@kavya-b I am not sure about using `$localize` in our library - sets an interesting precedent, and does this also assume that the application developer using our library will be using Angular's built in localization rather than an alternative tool such as ngx-translate? I need to do some more research on that as I'm not sure.  Perhaps rather than assuming the developer needs to use English and providing an English default but give them a backup option via `$translate`, we need to make no assumptions but still provide a way for the application developer to handle their own accessibility and language requirements.

Reply:
@mikerodonnell89 using `$localize` in fundamental ngx would be similar to using `i18n` markers, which we are already using in ngx, so the questions you pose are equally applicable to the usage of `i18n` markers which also assume English as the default currently. 
I am also not sure I have the answers to your questions; I thought fundamental ngx would be supporting Angular's i18n by default and migrating to any other translation/localization tool would be upto the end-user. Therefore, this probably needs to be part of a bigger discussion involving more people.
@droshev @InnaAtanasova @dimamarksman @fkolar @KevinOkamoto @Betrozov @manjunathanagaraj @platon-rov  what are your thoughts on this?

Reply:
For ngx-platform, we had a detailed discussion before rolling out the translation guidelines and publishing it in github wiki. We decided to use i18n directive for the hardcoded strings we introduce in any of the components and expect the app developers to use i18n extraction command to externalise the strings to be translated into the xlf file and then they proceed with providing values for translations.
There could be many other translation tools, but we decided to go with angular's i18n.
I hope the same can be extended to ngx-core as well so that both core and platform libraries have similar architecture for translation and it becomes easy for app developers to mix and match components from both core and platform.

If app developers decide to chose any other translation tool, they need to handle it on their own.

@droshev and team, your thoughts?

Reply:
Sending this for Core team discussion Monday. Apologies for the delay, we need to discuss it first. It's good that you had a discussion for platform, but the current PR is in Core so we need to see all the options and then come back with an answer.

Reply:
Hi there, we had a discussion on that, then I've tried to investigate how it can be done in a different (without `$localize`) way and I find your approach the most suitable here.

But we're not going to force customers to use the Angular i18n package, so this string should be getting from the @Input property and described in the docs so the customers will decide which localization package they want to use. 

cc @droshev @mikerodonnell89

Reply:
Yes, so we are fine using Live Announcer. 
We think we should not be opinionated with i18n. Instead, we can offer inputs with default values. We want to avoided hardcoded texts. @kavya-b can you introduce the right inputs for the texts?

Reply:
@mikerodonnell89 @droshev resolving this conversation as the changes have been made in accordance to the comments received.